autosarfactory/autosar_ui.py

```diff
@@ -65,32 +68,28 @@ def __initialize_ui(self):
         #print(style.themes)
 #        ['yaru', 'default', 'vista', 'classic', 'scidgreen', 'equilux', 'scidgrey', 'adapta', 'scidpink', 'scidmint', 'plastik', 'alt', 'clearlooks', 'itft1', 'smog', 'clam', 'scidsand', 'kroc', 'radiance', 'black', 'blue', 'arc', 'winxpblue', 'scidblue', 'ubuntu', 'keramik', 'winnative', 'elegance', 'aquativo', 'scidpurple', 'xpnative', 'breeze']
         #style.theme_use("equilux") #- very slow for some reason
-
         style.theme_use(self.__current_theme)
 
         # create ui components
-
         menubar = Menu(self.__root)
         filemenu = Menu(menubar, tearoff=0)
 
         menubar.add_cascade(label="Select Theme", menu=filemenu)
 

```

Comment:
Yes, I should them made the entries with capital letters from the beginning; makes the menu look better.